### PR TITLE
updates telegram prefixing with symbol

### PR DESCRIPTION
**Pull request by Nathan-Notifi**
https://user-images.githubusercontent.com/105258726/220763538-130eb42e-87d0-488c-af3a-9219c88480f1.mov

### CODE REVIEW

File: packages/notifi-react-card/lib/utils/stringUtils.ts [hunk 1]
```diff
@@ -10,3 +10,18 @@ export function addressEllipsis(
...
+  }
+  return telegramId;
+}
+
+export function prefixCharacter(value: string, character: string) {
+  return character + value;
+}
+
+export function prefixTelegramWithSymbol(telegramId: string) {
+  return prefixCharacter(telegramId, '@');
```
Comment (juni-notifi) [inline packages/notifi-react-card/lib/utils/stringUtils.ts:26]: should we also check that it doesn't already startWith('@')

File: packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts [hunk 2]
```diff
@@ -472,12 +484,16 @@ export const useNotifiSubscribe: ({
...
+    const finalTelegramId =
+      formTelegram === '' ? null : formatTelegramForSubscription(formTelegram);
+
     let finalPhoneNumber = null;
 
     if (isValidPhoneNumber(formPhoneNumber)) {
       finalPhoneNumber = formPhoneNumber;
     }
+
+    console.log('finalTelegram', finalTelegramId);
```
Comment (juni-notifi) [inline packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts:496]: let's remove logs

**Review by juni-notifi — APPROVED** [review]

File: packages/notifi-react-card/lib/utils/stringUtils.ts (hunk 1 above)
Comment (Nathan-Notifi) [reply packages/notifi-react-card/lib/utils/stringUtils.ts:26]: goot catch

File: packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts (hunk 2 above)
Comment (Nathan-Notifi) [reply packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts:496]: good catch

**Pull request merged**